### PR TITLE
pass local user to send local notifs

### PR DESCRIPTION
**Pull request by abdel-m**

### CODE REVIEW

File: crates/api_crud/src/comment/create.rs [hunk 1]
```diff
@@ -134,6 +134,7 @@ pub async fn create_comment(
     &local_user_view.person,
     true,
     &context,
+    Some(&local_user_view),
```
Comment (dullbananas) [inline crates/api_crud/src/comment/create.rs:137]: Do this for the other 3 api_crud files instead of using `None` in those.

**Review by dullbananas — CHANGES_REQUESTED** [review]
Still need to run `cargo +nightly fmt`

File: crates/api_crud/src/comment/create.rs (hunk 1 above)
Comment (abdel-m) [reply crates/api_crud/src/comment/create.rs:137]: Done

**Review by dessalines — CHANGES_REQUESTED** [review]
Some weird things like permissions changes going on with this PR.

Also, why is this necessary?

**Comment by abdel-m** [comment]
Sorry, I don't change any permissions myself. I ran the command cargo nightly and pushed a new commit with these changes. Do you want me to revert it?

**Comment by dessalines** [comment]
2 things, you need to change the permissions back to what they were, and you need to run nightly fmt, because the CI still shows it as failing a format check.

**Comment by abdel-m** [comment]
Could you please clarify which permissions you are referring to?

**Comment by dessalines** [comment]
Check your diff, looks like several files: create, read, update: https://github.com/LemmyNet/lemmy/pull/4920/files#diff-9684f2aae248152f65522320d71750b467c2904d34fc4515741cf211f11513e1

**Comment by dessalines** [comment]
BTW you are correct, this PR is necessary, because if there's no `local_user` passed to `CommentView::read(...`, then it the build_response function won't find any local community comments.

**Comment by dullbananas** [comment]
Run `rustup update` and try `cargo +nightly fmt` again

**Comment by dessalines** [comment]
They don't seem to understand, so I'll fix this one up.

**Review by dessalines — APPROVED** [review]

**Review by Nutomic — APPROVED** [review]

**Pull request merged**